lib/x11utils.pm

```diff
@@ -78,12 +79,19 @@ sub ensure_unlocked_desktop {
         my @tags = qw(displaymanager displaymanager-password-prompt generic-desktop screenlock screenlock-password authentication-required-user-settings authentication-required-modify-system guest-disabled-display oh-no-something-has-gone-wrong);
         push(@tags, 'blackscreen') if get_var("DESKTOP") =~ /minimalx|xfce/;    # Only xscreensaver and xfce have a blackscreen as screenlock
         push(@tags, 'gnome-activities') if check_var('DESKTOP', 'gnome');
-        assert_screen \@tags, no_wait => 1;
+        push(@tags, 'gnome-activities') if (!check_var('DESKTOP', 'gnome') && get_var("FIPS_ENABLED") && is_pvm);
+        # For powerVM x11 access in FIPs mode, we can connect it via vnc even in textmode
+        # Add some wait time for powerVM due to performance issue
+        my $timeout = is_pvm ? '120' : '30';
+        assert_screen(\@tags, timeout => $timeout, no_wait => 1);
         # Starting with GNOME 40, upon login, the activities screen is open (assuming the
         # user will want to start something. For openQA, we simply press 'esc' to close
         # it again and really end up on the desktop
         if (match_has_tag('gnome-activities')) {
             send_key 'esc';
+            # Add some more re-try for powerVM due to performance issue
+            send_key 'esc' if (get_var("FIPS_ENABLED") && is_pvm);
+            wait_still_screen 5 if (get_var("FIPS_ENABLED") && is_pvm);
```

Comment:
Same condition, why need separate?

Reply:
Good question, yutao!

I did many iterations's tests, and found that there still be some performance issue on powervm setup, so add some workaround here.